src/main/java/com/ProjectEureka/backend/controllers/VideoCommunicationController.java

```diff
@@ -0,0 +1,32 @@
+package com.ProjectEureka.backend.controllers;
+import com.ProjectEureka.backend.services.VideoCallService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collections;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/v1")
+public class VideoCommunicationController {
+
+    @Autowired
+    private VideoCallService videoCallService;
+
+    @GetMapping("call/{call_id}")
```

Comment:
Can you consider making these routes: `/agora/call`, `/agora/answer`, and `/agora/hangup`?